### ISSUE TITLE
FrontlineSMS closing, direct interaction with app?

### ISSUE THREAD

**Issue by poVoq** [thecartercenter/nemo#882]
See: https://www.frontlinesms.com/blog/2021/6/28/frontlinesms-is-closing

Maybe it would be possible to add an option to directly interact with the SMS sync Android app instead?

**Comment by cooperka** [comment]
hi @poVoq, maybe this can meet your needs? https://getnemo.readthedocs.io/en/latest/content/smssetup/smssetup.html#generic-sms

**Comment by poVoq** [comment]
Yes I saw this, but it seems to be a bit difficult to get working compared to the SMS sync Android app from FrontlineSMS.

**Comment by cooperka** [comment]
got it. this is likely not on our short-term roadmap, but we'd welcome a PR submission if you're interested in contributing!

**Comment by poVoq** [comment]
Another option would be to support webhooks that common messengers use. For example via Telegram bots or Slack / MS Teams it would quite easy to notify team members via such a webhook broadcast.

**Comment by cooperka** [comment]
we also need to support basic SMS options for locations where there's no data connection available, but webhooks also sounds like a great feature in general. I can imagine a lot of different uses for that.

**Comment by poVoq** [comment]
https://github.com/umer0586/AndroidSMSServer

Looks like it might be easy to support at least for sending SMS. If this could be configured per mission it would be quite easy to have a local Android device in country to send out SMS notifications.
